packages/cli/src/daemon-config.ts

```diff
@@ -242,10 +243,21 @@ export class IndexingConfig {
   })
   allowQueriesBeforeHistoricalSync = false
 
+  /**
+   * Allow Ceramic node to run in stand-along mode without Compose DB enabled
+   */
   @jsonMember(Boolean, {
     name: 'disable-composedb',
   })
   disableComposedb = false
+
+  /**
+   * Enable Historical data sync worker for Compose DB indexing
+   */
+  @jsonMember(Boolean, {
+    name: 'run-historical-sync-worker',
```

Comment:
is the word "worker" adding any clarity here?  How about just `run-historical-sync` or `enable-historical-sync`?